pennylane/qnn/keras.py

```diff
@@ -32,7 +32,8 @@
 
 
 class KerasLayer(Layer):
-    """Converts a :func:`~.QNode` to a Keras
+    """KerasLayer(qnode, weight_shapes: dict, output_dim, weight_specs: Optional[dict] = None, **kwargs)
```

Comment:
Note this line - for some reason it was showing as `KerasLayer(*args, **kwargs)`